packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss

```diff
@@ -23,7 +23,7 @@
       height: var(--default-button-height);
       padding: 0;
       background-color: var(--black20);
-      border: $border-black10;
+      border: 0px solid var(--black20);
```

Comment:
Why did you add the `solid var(--black20)` part?

Reply:
Even after the border has been made to 0px, a very thin line still exists as divider when observed by zooming in. so I have added `solid var(--black20)` to completely disable the divider.

Reply:
Are you sure? I can see that line
Only 0px:
<img width="842" alt="Screen Shot 2022-05-23 at 10 30 18" src="https://user-images.githubusercontent.com/61761153/169766799-45d37301-b195-4c4e-96ea-a5b84b498724.png">

Regular:
<img width="842" alt="Screen Shot 2022-05-23 at 10 30 52" src="https://user-images.githubusercontent.com/61761153/169766887-416c4c2f-d587-4300-9872-49fe6786cf93.png">

With `0px solid var(--black20)`
<img width="842" alt="Screen Shot 2022-05-23 at 10 32 00" src="https://user-images.githubusercontent.com/61761153/169767057-0632366a-af7f-4b68-a992-064363ae375b.png">

Reply:
You mean you *cannot see that line with only `0px`?

Reply:
With only `0px`
![Screenshot 2022-05-23 134134](https://user-images.githubusercontent.com/84218194/169774470-95f615fe-7a2b-4c71-8b17-f0194b2b3661.png)

Reply:
Please zoom in the above attached image, you can find a very thin line separating all roles and granular

Reply:
Ok so we will merge it like that with the solid